Source/ProjectFirma.Web/Views/Vendor/SearchVendor.cs

```diff
@@ -24,16 +24,16 @@
 
 namespace ProjectFirma.Web.Views.Vendor
 {
-    public abstract class EditVendor : LtInfo.Common.Mvc.TypedWebPartialViewPage<EditVendorViewData, IEditVendorViewModel>
+    public abstract class SearchVendor : LtInfo.Common.Mvc.TypedWebPartialViewPage<SearchVendorViewData, ISearchVendorViewModel>

```

Comment:
This wasn't even necessary since we weren't going to be editing Vendors for this first card of viewing the imported data, but it came up when we were using a different entity's Detail view as a template for Vendor's counterpart.

Well, I imagine that this will need to have been done at some point, anyway.